docs/docs/widgets/textarea.md

```diff
@@ -52,37 +53,37 @@ In the input field under **Tooltip**, you can enter some text and the component
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
-## Layout
+## Devices
 
-| <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
-|:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div>                                                            |
+| :--------------------------------------------- | :------------------------------------------------ | :-------------------------------------------------------------------------------------------------------------- |
+| Show on desktop                                | Toggle on or off to display desktop view.         | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}` |
```

Comment:
Hey @mihir2004 you need to update the content of expected value column from the reference given to you in issue.

Reply:
Hey I did not understand this. Do I need to replace to content and put In that of text input ?.

Reply:
Hey, please update the content of the expected value column exactly as given in the text input docs, i.e. instead of ` You can programmatically determining the value by clicking on **fx** to set the value {{true}} or {{false}} ` we need `You can set it with the toggle button or dynamically configure the value by clicking on fx and entering a logical expression.` there.
let me know if you need any further assistance, thanks!